Update package.json to latest versions

Cruikshanks opened the pull request:
Dependabot has been keeping the project up to date. But as we're now ready to start using the project in earnest we felt it a could time to update `package.json` to match.

fayaz0211 commented on features/internal/manage/notifications/return_invitations.feature:
```diff
@@ -0,0 +1,35 @@
+    And I can confirm sending the returns invitations
+    And I can see the returns invitations success page
+
+  Scenario: User can send returns invitations excluding some licences
```
We can have these scenarios into one as a Scenario Outline as the only difference is two different data  "no" and "some" and the rest of the steps are same.

fayaz0211 commented on features/internal/manage/notifications/return_invitations.feature:
```diff
@@ -0,0 +1,35 @@
+      | wirs                |
+      | nps                 |
+      | psc                 |
+      | environment_officer |
```
may be nice to have a new line after  the end of the user table

fayaz0211 commented on features/internal/manage/notifications/return_invitations.feature:
```diff
@@ -0,0 +1,35 @@
+  Scenario Outline: Users without billing and data permission can't send returns invitations
+    Given I logged in as <user> user
+    And I navigate to the "Manage" section
+    Then I can't see the "returns invitations" link as <user>
```
I don't think we need to say the user type explicitly again in the Then step to check the outcome as the Given step itself gives the user information anyway. Suggest to remove it and have it as
 Then I can't see the "returns invitations" link

fayaz0211 commented on features/step_definitions/internal/notifications/common_steps.rb:
```diff
@@ -0,0 +1,21 @@
+    "returns invitations" => "Invitations"
+  }
+  page = Pages::Internal::ManagePage.new
+  expect(page).to have_no_text("Send return notices") unless user.eql? "wirs"
```
I think its nice to check only the wirs user can able to see the option but for this test being  for invitations visibility to a particular types of users  it be good if we could exclude wirs user also like other users in the scenario.

Cruikshanks committed:
Update package.json to latest versions

Dependabot has been keeping the project up to date. But as we're now ready to start using the project in earnest we felt it a could time to update `package.json` to match.
